platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/RichTextEditorState.kt

```diff
@@ -30,21 +32,21 @@ class RichTextEditorState internal constructor() {
      *
      * @param inlineFormat which format to toggle (e.g. [InlineFormat.Bold])
      */
-    fun toggleInlineFormat(inlineFormat: InlineFormat) {
+    suspend fun toggleInlineFormat(inlineFormat: InlineFormat) = withContext(Dispatchers.Main) {
```

Comment:
Why this change? Is it related to the dependency updates?

Reply:
This is related to an issue with tests since updating compose. The UI tests are failing with:

```
android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
```

This change makes the problem go away but I'm not sure it is the right solution.

Reply:
If `state.toggleInlineFormat()` is not called from the main thread in a non-test environment, then this protects the caller from causing a crash.

But that's not possible or not something we care about, we can just ensure tests call these functions on the main thread.

Reply:
Indeed. It is very much preferable that our public API is not dictated by test necessities. Better change the tests.